test/integration/cloudkms_test_mixins_v1.yaml

```diff
@@ -0,0 +1,76 @@
+type: google.api.Service
+config_version: 3
+name: cloudkms.googleapis.com
+title: Cloud Key Management Service (KMS) API
+
+apis:
```

Comment:
Do you think we could add `google.cloud.location.Locations` to this? KMS includes it on the internal Service config, so it would be part of the service_yaml eventually.

Reply:
Leaving it as just IAM for now, since the public java-kms repo doesn't seem to have Location RPCs in the generated surface yet.